tools/generator/src/DTO/XcodeScheme.swift

```diff
@@ -237,7 +237,7 @@ No `BazelLabel` values were provided to `XcodeScheme.TestAction`.
 
 extension XcodeScheme.TestAction: Decodable {
     enum CodingKeys: String, CodingKey {
-        case buildConfigurationName
+        case buildConfigurationName = "buildConfiguration"
```

Comment:
`buildConfigurationName` is the name that `XcodeProj` uses, but in Starlark I felt it was better as `build_configuration`. With #1794 this would just be `"b"`.